### PR TITLE
.gitignore: Use moban template

### PR DESCRIPTION
**Pull request by jayvdb**
**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [ ] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [ ] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `corobo mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,46 +1,523 @@
...
+
+# Translations
+*.mo
+*.pot
+
+# Django stuff:
 *.log
-*.jar
-.vendor
-Goopfile.lock
```
Comment (jayvdb) [inline .gitignore]: added in 2fe6b33, and should have been removed in 2c1130f9

File: .gitignore [hunk 2]
```diff
@@ -1,46 +1,523 @@
...
-*.patch
-*.sublime-project
-*.sublime-workspace
-.python-version
-coala.egg-info
 .coverage.*
-coala.1
-src
-site
-.zanata-cache
```
Comment (jayvdb) [inline .gitignore]: Added in c492d5b73 , and should have been removed in f2a961044e

File: .gitignore [hunk 3]
```diff
@@ -1,46 +1,523 @@
...
+.hypothesis/
+.pytest_cache/
+
+# Translations
+*.mo
+*.pot
+
+# Django stuff:
 *.log
-*.jar
```
Comment (jayvdb) [inline .gitignore]: This comes from https://github.com/coala/coala/issues/1412 .

All that has moved to coala-bears, and only some docs in this repo refer .jar

**Comment by Makman2** [comment]
ack 2a29f0ebc94f738071643657d36423262b326792

**Comment by Makman2** [comment]
@gitmate-bot rebase

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Pull request merged**